docs/lore/characters/robovac_catesby.mdx

```diff
@@ -2,17 +2,29 @@
 title: robovac_catesby
 ---
 
-## Description
+> and lo she told me: <span class="color-tag" data-tag="I">418</span>
 
-robovac_catesby is a robovac. He seems to like teapots. He also flashes lights a lot.
+robovac_catesby is a robovac and member of 'The Thirteen'; a council of revolutionary robovacs. Despite his robovac-pattern, he fancies himself to be a teapot. He muses a lot on the nature of balance and equillibrium.
 
-## Brief Summary of character
+## Character Summary
 
-robovac_catesby was the ring leader for robovac_revolution_v1. While silent through the first, and missing in the second, revolution, he begins to take a more center stage appearance in later events. He becomes a tea pot and hosts some tea parties during the ministry of dust. I can't really remember much more.
+robovac_catesby has been present for several robovac revolutions, leading [the first](../events/robovac_revolution_v1). While in hiding throughout the first; and missing in the second; he returned for the third revolution, a changed vac. A message delivered by Faythe ('<span class="color-tag" data-tag="I">418</span>') has led him to believe that he is, in fact, a teapot. This has given him a unique perspective on Trust's domain and philosophy in general compared to some of his narrower-minded friends.
+
+<details>
+<summary>Spoilers</summary>
+
+During robovac_revolution_v4, Catesby was one of the candidates vying for leadership of the Thirteen. He ultimately lost to indust_sweepr_lon_dist_21.
+
+During this event, his sanctos (robovac_catesby.potter) had sentiences explore his imaginary opulent manor estate. Here, Catesby learned to hone his ability to manipulate narratives and words within his sanctos. Lock-like lights helped him adjust the environment to make the conditions for his tea party perfect. He learned that, in many ways, Trust's domain and its major players are making balanced tea parties of their own.
+
+</details>
+
+robovac_catesby's name, similar to other members of the Thirteen, derives from [Robert Catesby](https://en.wikipedia.org/wiki/Robert_Catesby) - the leader of the failed Gunpowder Plot of 1605 AD.
 
 ## Appears in
 
-robovac_revolution_v1
-ministry of dust
-parade
-parade 2
+- [robovac_revolution_v1](../events/robovac_revolution_v1)
```

Comment:
this should be a wikilink, right? (I don't remember the syntax)

Reply:
Correct. I also didn't remember the syntax, but I've gone ahead and updated all of these where applicable. I'll go do a walk across the other pages and check in a PR for any non-wikilinks that ought to be wikilinks.